src/data_info/routes/avus.clj

```diff
@@ -13,7 +12,6 @@
     :tags ["data-by-id"]
 
     (GET "/metadata" [:as {uri :uri}]
-      :middleware [otel-middleware]
```

Comment:
I can't lie, I'm really glad to see all these individual middlewares that had to be added to make it work, go away.